image/image.go

```diff
@@ -8,10 +8,24 @@ import (
 // Type must be a "Backing file format name string" that appears in QCOW2.
 type Type string
 
+type Extent struct {
+	// Offset from start of the image in bytes.
+	Start int64 `json:"start"`
```

Comment:
This is not strictly needed, since when calling Next(start, length) we always return the requested start value.

Reply:
Removing Start makes testing easier and eliminate the error of invalid extents list. When consuming the extents we keep current offset externally so we don't really need it. When printing extents having start is helpful. If we don't report it we have to add another type with Start for encoding json.